homeassistant/components/flume/sensor.py

```diff
@@ -143,4 +143,4 @@ def native_value(self):
 
 
 def _format_state_value(value):
-    return round(value, 1) if isinstance(value, Number) else None
+    return round(value, 2) if isinstance(value, Number) else None
```

Comment:
We shouldn't format any state value anymore. Home Assistant has built-in support for rounding and we should return the raw value. See `suggested_precision` in docs https://developers.home-assistant.io/docs/core/entity/sensor#properties

Reply:
Thanks! This definitely makes this more flexible for users to go as granular as they want in this manner as can be seen here:  
<img width="504" alt="Screenshot 2023-06-25 at 6 41 00 PM" src="https://github.com/home-assistant/core/assets/9800912/d22824ee-1b4d-4888-9437-ad46e7ff1df7">